src/Caching/SqlServer/src/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
Add `#nullable enable`.

The right way to create these files is _not_ in Visual Studio. Instead, copy eng/PublicAPI.empty.txt to PublicAPI.Shipped.txt and PublicAPI.Unshipped.txt in new projects and then fill them up in Visual Studio.